src/Chainweb/Utils.hs

```diff
@@ -1405,14 +1373,6 @@ showIpv6 ha = T.intercalate ":"
   where
     (a0,a1,a2,a3,a4,a5,a6,a7) = hostAddress6ToTuple ha
 
-#if !MIN_VERSION_network(3,0,0)
```

Comment:
I noticed that NFData instance in network, version 3.1.1.1, is slightly different than what you had previously. Is this desired?

Reply:
hmm, in any case whatever is in the network package is authoritative.